From 62ad4dfa328dc24b4786d9c0c35ebcbd1fe574b9 Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Thu, 2 Jul 2015 14:30:09 -0400
Subject: [PATCH 40/41] Make the grub1 "defaultkernel" test more reliable.

Basically just do it twice with various incarnations of
--boot-filesystem={/,/boot}

Related: rhbz#1184014

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 test.sh                        | 3 ++-
 test/results/defaultkernel/g.2 | 1 +
 2 files changed, 3 insertions(+), 1 deletion(-)
 create mode 100644 test/results/defaultkernel/g.2

diff --git a/test.sh b/test.sh
index efd9e84..74857af 100755
--- a/test.sh
+++ b/test.sh
@@ -299,7 +299,8 @@ grubDisplayTest grub.10 defaulttitle/g.10 --default-title
 grubDisplayTest grub.11 defaulttitle/g.11 --default-title
 
 testing="GRUB display default kernel"
-grubDisplayTest grub.1 defaultkernel/g.1 --default-kernel
+grubDisplayTest grub.1 defaultkernel/g.1 --boot-filesystem=/boot --default-kernel
+grubDisplayTest grub.1 defaultkernel/g.2 --boot-filesystem=/ --default-kernel
 
 testing="LILO default directive"
 liloTest lilo.1 default/l1.1 --set-default=/boot/vmlinuz-2.4.18-4
diff --git a/test/results/defaultkernel/g.2 b/test/results/defaultkernel/g.2
new file mode 100644
index 0000000..d15855e
--- /dev/null
+++ b/test/results/defaultkernel/g.2
@@ -0,0 +1 @@
+/vmlinuz-2.4.7-2
-- 
2.4.3

